packages/angular/src/errorhandler.ts

```diff
@@ -0,0 +1,118 @@
+import { HttpErrorResponse } from '@angular/common/http';
+import { ErrorHandler as AngularErrorHandler, Injectable } from '@angular/core';
+import * as Sentry from '@sentry/browser';
+
+/**
+ * Options used to configure the bahavior of ErrorHandler.
+ */
+export interface ErrorHandlerOptions {
+  logErrors?: boolean;
+  showDialog?: boolean;
+  dialogOptions?: Sentry.ReportDialogOptions;
+  /**
+   * Custom implementation of error extraction from the raw value captured by the Angular.
+   * @param error Value captured by Angular's ErrorHandler provider
+   * @param defaultExtractor Default implementation that can be used as the fallback in case of custom implementation
+   */
+  extractor?(error: unknown, defaultExtractor: (error: unknown) => unknown): unknown;
+}
+
+/**
+ * Implementation of Angular's ErrorHandler provider that can be used as a drop-in replacement for the stock one.
+ */
+@Injectable({ providedIn: 'root' })
+class SentryErrorHandler implements AngularErrorHandler {
+  private readonly _options: ErrorHandlerOptions;
+
+  public constructor(options?: ErrorHandlerOptions) {
+    this._options = {
+      logErrors: true,
+      ...options,
+    };
+  }
+
+  /**
+   * Method called for every value captured through the ErrorHandler
+   */
+  public handleError(error: unknown): void {
+    const extractedError = this._extractError(error) || 'Handled unknown error';
+
+    // Capture handled exception and send it to Sentry.
+    const eventId = Sentry.captureException(extractedError);
+
+    // When in development mode, log the error to console for immediate feedback.
+    if (this._options.logErrors) {
+      console.error(extractedError);
```

Comment:
Why not use the `logger` util?

Reply:
Because it's to preserve original Angular's behavior. Same as we have in Vue - https://github.com/getsentry/sentry-javascript/blob/master/packages/integrations/src/vue.ts#L405-L410